src/style.css

```diff
@@ -86,11 +86,22 @@ li a {
   bottom: 0;
   color: white;
   padding: 10px;
-  width: 100%;
+  cursor: pointer;
+}
+
+.text > button:hover {
```

Comment:
**[OPTIONAL]** I've noticed that you are using most of the selectors this way instead of using classes, I recommend you using classes to avoid any future conflicts with other HTML elements and for easing the selection on yourself. 😉✌🏻